Sources/Orbit/Components/TileGroup.swift

```diff
@@ -2,6 +2,19 @@ import SwiftUI
 
 /// Wraps tiles to show related interactions.
 ///
+/// Only ``Tile``s should be used inside of a group.
+///
+/// Tiles in a group show separators by default. Use ``Tile/tileSeparator(_:)`` to suppress this behavior:
```

Comment:
For more confidence, we should emphasise here that caller does not need to manually solve the last separator. Something like:

```
Tiles in a group show separators by default, except for the last tile, which never shows a separator.
```

Reply:
Updated.